docs/openrpc.json

```diff
@@ -1444,6 +1444,23 @@
                     }
                 ]
             },
+            "BlockNumberOrTagOrHash": {
+                "title": "Block number or tag",
+                "oneOf": [
+                    {
+                        "title": "Block number",
+                        "$ref": "#/components/schemas/uint"
+                    },
+                    {
+                        "title": "Block tag",
+                        "$ref": "#/components/schemas/BlockTag"
+                    },
+                    {
+                        "title": "Block hash",
+                        "$ref": "#/components/schemas/hash32"
+                    }
```

Comment:
According to the [EIP 1898](https://eips.ethereum.org/EIPS/eip-1898) blockParam can either be a String or an Object. 
If is a String, it can either be a BlockTag or Number. 
ex: "earliest", "latest", "0x1"
If is an Object, it can contain a BlockNumber or a BlockHash Field. 
```json
{
   "blockHash": "0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3"
}

or

{ 
    "blockNumber":"0x1"
}
```

I don't think that it supports a blockHash in case of receiving a single string as is implied on this PR.

Reply:
@AlfredoG87 You are right about this. According to this EIP hash could be an object or string. Currently, we support blockParam for eth_call only, in other relays like Alchemy they also support blockHash as an object on eth_call and eth_estimateGas, but not on other endpoints. 
Not sure if we should introduce blockParam for all endpoints or jsut to follow alchemy and do it only for estimateGas
@Nana-EC What do you think?

Reply:
@konstantinabl I'd say keep it simple and targeted for now.
So in this PR focus on eth_getStorageAt issue and resolve it.
@AlfredoG87 I think you added the logic to handle string or object, we should make sure that is in a shared place so all endpoints can use it easily as need be.
It looks like the [EIP 1898](https://github.com/hashgraph/hedera-json-rpc-relay/issues/822) support wasn't completely resolved as we thought.
We can add it to each endpoint as we need so we're intentional about it.
Can someone re-open that ticket and update the Epic to capture all the applicable endpoints and the remaining work so be done (string or object support).
We can tackle the work in a future sprint.

Reply:
@Nana-EC thanks, im merging it then :)